### PR TITLE
Add Personal Info and Projects

### PR DESCRIPTION
**Pull request by aroquev00**
Added a personal introduction, photo, personal background and projects list.

No styling added yet, will add as soon as all info is on HTML file.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] PR has a descriptive title
- [x] PR contains closely related changes only (should not have many unrelated changes)
- [ ] STEP pod members have been added as reviewers
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

image

image

### CODE REVIEW

File: aroquev/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,90 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Armando Roque's Portfolio</h1>
+      <p>Hi and welcome to my personal portfolio! I am Armando, and here you will find interesting things about me (thanks for being interested!). I am currently studying Computer Science
+      at Tec de Monterrey. During the Summer of 2020 I will be working as a Google intern in the STEP program (this is where this project was born), really exciting stuff!</p>
+      <img id="main-photo" src="images/my-photo.JPG" alt="A photo of Armando">
+
+			<hr>
```
Comment (seanjohnite) [inline aroquev/src/main/webapp/index.html:16]: nit: Can you look into reformatting this HTML file? Maybe with 2-space tabs?

File: aroquev/src/main/webapp/script.js [hunk 2]
```diff
@@ -16,6 +16,7 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
+
```
Comment (seanjohnite) [inline aroquev/src/main/webapp/script.js:19]: line added here? nit: probably remove?

File: aroquev/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,8 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#main-photo {
+    width: 600px;
+    height: 400px;
+}
```
Comment (seanjohnite) [inline aroquev/src/main/webapp/style.css:14]: nit: add newline to end of file

Comment (aroquev00) [reply aroquev/src/main/webapp/style.css]: Done!

File: aroquev/src/main/webapp/script.js (hunk 2 above)
Comment (aroquev00) [reply aroquev/src/main/webapp/script.js]: Done.

File: aroquev/src/main/webapp/index.html (hunk 1 above)
Comment (aroquev00) [reply aroquev/src/main/webapp/index.html:16]: Done.

Comment (seanjohnite) [reply aroquev/src/main/webapp/index.html:16]: Seems like the <hr> above is kind of hanging out and messing with indenting for the rest of the file?

Comment (aroquev00) [reply aroquev/src/main/webapp/index.html:16]: Yep, just fixed that in the latest commit.

Comment (aroquev00) [reply aroquev/src/main/webapp/index.html:16]: Oh no, you are right. Why does indentation appear different on github?

**Pull request merged**